pontoon/base/models.py

```diff
@@ -472,6 +472,25 @@ def get_top_instances(self):
 @python_2_unicode_compatible
 class Locale(AggregatedStats):
     code = models.CharField(max_length=20, unique=True)
+
+    # Codes related to Microsoft products.
+    ms_translator_code = models.CharField(
+        max_length=20,
+        blank=True,
+        help_text="""
+        Microsoft Translator maintains its own list of <a href="https://msdn.microsoft.com/en-us/library/hh456380.aspx">supported locales</a>.
+        Choose a locale from that list that's is the closest match or leave it blank to disable support for this feature.
```

Comment:
...*to disable support for this feature* -> *to disable support for Microsoft Translator*